game.js

```diff
@@ -1430,12 +1430,17 @@ class GameManager extends EventTarget {
     return await this.handleDropUrlToPlayer(u, index);
   }
   async handleDropJsonToPlayer(j, index) {
+    const localPlayer = playersManager.getLocalPlayer();
+    localVector.copy(localPlayer.position);
+    if(localPlayer.avatar)
+      localVector.y -= localPlayer.avatar.height;
     const u = getDropUrl(j);
-    return await this.handleDropUrlToPlayer(u, index);
+    return await this.handleDropUrlToPlayer(u, index, localVector);
   }
-  async handleDropUrlToPlayer(u, index) {
+  async handleDropUrlToPlayer(u, index, position) {
```

Comment:
Index is not used so why not remove it? Also with these miscellaneous optional arguments we should use an options object rather than positional args.